src/main/java/com/jcabi/github/PullComment.java

```diff
@@ -77,6 +77,13 @@
     int number();
 
     /**
+     * Get its author.
+     * @return Pull comment author
+     * @throws IOException If there is any I/O problem
+     */
+    User author() throws IOException;
```

Comment:
@krzyk 
sorry, I don't get.... should I remove it from PullComment?

by doing so it will defeat this PR intention.
I probably missed something.

You are talking about this class, right:
https://github.com/velo/jcabi-github/blob/author/src/main/java/com/jcabi/github/PullComment.java#L93